src/sourmash/signature.py

```diff
@@ -192,6 +193,16 @@ def __reduce__(self):
             ),
         )
 
+    def __copy__(self):
+        a = SourmashSignature(
+            mh=self.minhash,
+            self.name,
```

Comment:
please see `tests/test_signature.py` for how to create a new `SourmashSignature` object.

Reply:
Okay